main.go

```diff
@@ -10,11 +10,14 @@ import (
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
-var username = flag.String("username", "", "username to authenticate to the Vigor")
-var password = flag.String("password", "", "password to authenticate to the Vigor")
-var host = flag.String("host", "", "hostname/ip the Vigor is reachable on")
+var (
+	username = flag.String("username", "", "username to authenticate to the Vigor")
+	password = flag.String("password", "", "password to authenticate to the Vigor")
+	host     = flag.String("host", "", "hostname/ip the Vigor is reachable on")
```

Comment:
do we want to use something like [https://github.com/alecthomas/kong], and also make the listen address configurable?

Reply:
In priciple yes, but I guess for this project a argument parses lib would be a bit overkill. We only have three parameter, all of them are simple strings and required. Nothing fancy that would justify adding such a big dependency imho

Reply:
Fair enough, I was just hoping on a bit more inspection. However, even `flag` supports a `--help`, so this should indeed be sufficient.